chrome/content/zotero/zoteroPane.js

```diff
@@ -2786,6 +2786,16 @@ var ZoteroPane = new function()
 				return;
 			}
 			
+			//Special treatment for zotero links
+			if (uri.match(/^zotero\:\/\/(open-pdf|select)/)) {
```

Comment:
There's no way that `.test` doesn't work here. Also, you forgot the trailing `\/`